tests/test_ingesters.py

```diff
@@ -308,6 +312,8 @@ def requests_get_side_effect(self, url, **kwargs):  # pylint: disable=unused-arg
         return response
 
     def setUp(self):
+        Parameter.objects.count = mock.Mock()
+        Parameter.objects.count.return_value = 2
```

Comment:
Please use patch for this, in order to limit unintended side effects

Reply:
Done in the next commit

Reply:
Done in the next commit